README.md

```diff
@@ -42,8 +60,8 @@ pip3 install -r requirements.txt
 
 ### Running examples
 - Change into directory for phase of project you're working with
-- Start the python backend with `uvicorn api:app --reload`
-- (For 4-6) run `npm install` then `npm start`
+- Starting with step 3, start the python backend with `uvicorn api:app --reload`
+- (For 4-6) run `npm install` then `npm run dev`
```

Comment:
I'd prefer the same phrasing between "Starting with step 3" and "(For 4-6)" but no preference as to which format is used.

Reply:
If you'd prefer the original phrasing (but with `npm run dev`) then that's fine. I can remove the "Starting with step 3, " text